lib/tasks/pipeline_monitor.rake

```diff
@@ -96,7 +98,19 @@ class CheckPipelineRuns
       'max_job_dispatch_lag_seconds' => PipelineRun::MAX_JOB_DISPATCH_LAG_SECONDS,
       'job_tag_prefix' => PipelineRun::JOB_TAG_PREFIX,
       'job_tag_keep_alive_seconds' => PipelineRun::JOB_TAG_KEEP_ALIVE_SECONDS,
-      'draining_tag' => PipelineRun::DRAINING_TAG
+      'draining_tag' => PipelineRun::DRAINING_TAG,
+      'batch_configurations' => [
+        {
+          'queue_name' => Sample::DEFAULT_QUEUE,
+          'vcpus' => Sample::DEFAULT_VCPUS,
```

Comment:
Technically we could add up the vcpus from each job description in those queues, and not need this param.   I'm fine with this, though.

Reply:
Yes, I just reused this constant since it was already there. That is saving extra calls to get the job definitions.